Cargo.toml

```diff
@@ -35,6 +35,7 @@ bytemuck = { version = "1.12.2", features = ["derive"] }
 [features]
 std = []
 example_generated = []
+serde = ["serde_core"]
```

Comment:
It looks like our MSRV doesn't like this, so we might need to use an alternative:

```toml
[dependencies]
serde = { version = "1.0.220", optional = true, default-features = false, package = "serde_core" }
```